RELEASE_NOTES.rst

```diff
@@ -9,8 +9,10 @@ ARTIQ-7
 Highlights:
 * WRPLL
 * ``get()``, ``get_mu()``, ``get_att()``, and ``get_att_mu()`` functions added for AD9910 and AD9912
+* Phaser coredevice driver: refactor, bug fixes and added documentation
 
 Breaking changes:
+* Updated Phaser-Upconverter default frequency configuration. The new default uses the design target PFD frequency
```

Comment:
Also breaking changes to sif_sync, oscillators, etc!

Reply:
Ack, breaking changes are:
* New LO frequency
* New Kasli oscillator state after `init`

Reply:
And the abstraction over and ownership of `sif_sync`.